### ISSUE TITLE
Lunar calculations not yet supported

### ISSUE THREAD

**Issue by bmorris3** [astropy/astroplan#35]
We need an [astropy issue to be addressed](https://github.com/astropy/astropy/issues/3920), which is currently being addressed by @eteq, before we can move forward on the moon phase, illumination, position calculations. 

Once this issue has been addressed, we can bring in this jplephem implementation of `get_moon` from [this toy notebook](https://github.com/bmorris3/gsoc2015/blob/master/jplephem.ipynb).

**Comment by cdeil** [comment]
I've put "whishlist" as milestone for this.
Probably we want astroplan 0.1 to work with astropy 1.0 and then only require astropy 1.1. later?

**Comment by eteq** [comment]
The issue @bmorris3 linked to here actually is not a show-stopper and can be worked around in astroplan... but there's another issue (not yet in the astropy issue tracker because I'm still determining it's exact nature) that will be harder to work around from the outside.  So for *that* issue I agree we'll need to think about how to match astroplan to astropy versions.  It may not be an issue if the fix ends up in a bug fix release (i.e. astropy 1.0.4) because then we can just require that.

**Comment by bsipocz** [comment]
Since astropy/astropy#4890 is merged this can be closed I think.

**Comment by bmorris3** [comment]
This one can be closed when we actually implement the lunar features `moon_altaz`, `MoonSeparationConstraint`, etc using astropy rather than PyEphem.

**Comment by bmorris3** [comment]
This is closed by #167.
